contrib/mesos/ci/run-with-cluster.sh

```diff
@@ -79,6 +79,7 @@ exec docker run \
   -ceux "\
     make clean all && \
     trap 'timeout 5m ./cluster/kube-down.sh' EXIT && \
+    ./cluster/kube-down.sh && \
```

Comment:
this should actually go before the trap. we don't want to down twice on failure or sigint